bitnami/common/Chart.yaml

```diff
@@ -20,4 +20,4 @@ sources:
   - https://github.com/bitnami/charts
   - http://www.bitnami.com/
 type: library
-version: 1.4.3
+version: 2.1.0
```

Comment:
Please bump the minor version, as not it is not adding a breaking change

Reply:
Done